cgi-bin/LJ/Web.pm

```diff
@@ -2017,26 +2026,52 @@ sub entry_form_postto_widget {
 
     return undef unless $res;
 
-    my @journals = map { $_, $_ } @{$res->{'usejournals'}};
+    LJ::need_res( { group => 'jquery' }, 'js/quickupdate.js' );
+
+    my @journals = map { {
+            value => $_,
+            text => $_,
+            data => { minsecurity => minsec_for_user( $_ ), iscomm => 1 }
+        } } @{$res->{'usejournals'}};
```

Comment:
Style: no optional quotes (such as around most keys), so `usejournals` alone is fine.